drivers/include/fxos8700.h

```diff
@@ -47,8 +47,8 @@ extern "C" {
  * If set to 0, measurements will be converted to mg.
  * If set to 1, raw adc readings will be returned.
  */
```

Comment:
Here documentation can be adapted to:
```C
/**
 * @brief   Default raw value mode for accelerator
 *
 * Set to 1 to return raw ADC readings. Otherwise measurements be converted to
 * mg.
 */
```

Reply:
Done. 
Doxygen build works fine.